pom.xml

```diff
@@ -96,6 +96,15 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
         <version>${maven-jar-plugin.version}</version>
+        <configuration>
+          <archive>
+            <manifest>
+              <addClasspath>true</addClasspath>
+              <classpathPrefix>lib/</classpathPrefix>
+              <mainClass>com.github.yuiskw.beam.BigQuery2Datastore</mainClass>
+            </manifest>
+          </archive>
+        </configuration>
```

Comment:
Can you remove the part? Because it would be nice to separate supporting another timestamp format from specifing the main class.

Reply:
Yes just removed